yt_dlp/extractor/wrestleuniverse.py

```diff
@@ -261,7 +266,7 @@ class WrestleUniversePPVIE(WrestleUniverseBaseIE):
 
     def _real_extract(self, url):
         lang, video_id = self._match_valid_url(url).group('lang', 'id')
-        metadata = self._download_metadata(url, video_id, lang, 'eventFallbackData')
+        metadata = self._download_metadata(url, video_id, lang, ('eventFallbackData',))
```

Comment:
revert after applying suggestion for `_download_metadata`
```suggestion
        metadata = self._download_metadata(url, video_id, lang, 'eventFallbackData')
```